Makefile

```
@@ -11,12 +11,16 @@ release:
 	cargo build -p near-vm-runner-standalone --release
 	cargo build -p state-viewer --release
 	cargo build -p store-validator --release
+	cargo build -p param-estimator --release
```

```suggestion
	cargo build -p runtime-params-estimator --release
```
Sorry I told you incorrect package name, could you replace it here and below?

---

updated.